DeepSpeech.ipynb

```diff
@@ -616,7 +616,9 @@
     "    batch_x, batch_y, n_steps = batch_set.next_batch()\n",
     "\n",
     "    # Set batch_seq_len for the batch\n",
-    "    batch_seq_len = batch_x.shape[0] * [n_steps]\n",
+    "    multiple = tf.slice(tf.shape(batch_x),[0],[1])\n",
+    "    tile = tf.slice(tf.shape(batch_x),[1],[1])\n",
+    "    batch_seq_len = tf.tile(tile, multiple)\n",
```

Comment:
I have a hard time following this, was there any reason to do this instead of just batch_x.shape[0] * [batch_x.shape[1]]? It would help to have a comment on the expected shape of batch_x here to understand what's happening.